### ISSUE TITLE
Deleted script still functioning

### ISSUE THREAD

**Issue by Czensored** [Tampermonkey/tampermonkey#657]
(Please fill out the issue template with your details)

## Expected Behavior
When I delete a script it should terminate all processes. I am not that experienced at coding and I do not know if this is true.

## Actual Behavior
<img width="374" alt="screen shot 2019-02-27 at 1 08 50 pm" src="https://user-images.githubusercontent.com/37640727/53523135-db45d800-3a90-11e9-96cd-eb261b9fc634.png">

## Specifications

- Chromium: (i.e. 64.0.3282.140)
- TM: (i.e. 4.5.5648)
- OS: (i.e. MacOS 10.13.4)

## Script
I do not remember what script was deleted
(Please give an example of the script if applicable.)

```js
// ==UserScript==
// @name MyName
// @namespace http://mywebsite.com
// @Version 1.0
// @description try to take over the world!
// @author Me
// @match http://mywebsite.com/*
// @require http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js
// @include https://mywebsite.com/*
// @grant none
// ==/UserScript==
```

**Comment by derjanb** [comment]
This is not possible without reloading the/all tabs, which might cause user data loss.
